WalletWasabi/Services/WalletService.cs

```diff
@@ -753,7 +753,11 @@ private async Task<Block> TryDownloadBlockFromLocalNodeAsync(uint256 hash, Cance
 						{
 							ConnectCancellation = handshakeTimeout.Token,
 							IsRelay = false,
-							UserAgent = $"/Wasabi:{Constants.ClientVersion}/"
+#if RELEASE
+							UserAgent = $"/Wasabi:{Constants.ClientVersion.ToString(3)}/"
+#else
+							UserAgent = $"/Wasabi:{Constants.ClientVersion.ToString(4)}/"
+#endif
```

Comment:
Shouldn't we put what Bitcoin Core puts here?

Reply:
This is only for when we connect to our own node